### DISCUSSION TITLE
Is anyone interested in a commented fennel (lisp) based config?

### DISCUSSION THREAD

**Discussion by erikLundstedt** [endaaman/tym#97] [General]
I've been using tym for quite a while now and I've got a pretty good config,
However, I am one of those people who like writing the lisp family of languages, so I'm using something called fennel for my configuration (it's a lisp dialect that can be used instead of Lua).

My question is if anyone would be interested in seeing my configs, complete with comments and explanations of what everything does,
It's currently part of my configs repository on gitlab but I think it could be useful for others to see, I could upload it on GitHub or maybe make a new gitlab repo, and also make more and better comments.
I could also upload the compiled(transpiled) Lua code.

Please let me know if there is any interest.

**Comment by endaaman** [comment]
Not only configs in lisp, I think useful techniques should be shared among users as well. Experimentally, I made the wiki editable by anyone. Now you can write some tips there.

I also migrated Pro tips section to Wiki. I was feeling the section was not read well or I wondered if it should necessarily be included in README, It was a good opportunity to get organized.

**Reply by erikLundstedt** [reply]
thanks I will add and maybe "show off" my configs once I get time to clean them out a little(im breaking several "[lisp-style-rules](https://fennel-lang.org/style)" in my own code, by habit) and write some comments

I will probably end up placing a link(probably near the bottom) to the right folder of my dotfiles repo so people who want to can try out the latest version



also, im actually using the example code to zoom etc(except translated to fennel)

**Reply by bhougland18** [reply]
I would love to see your fennel config.
